### DISCUSSION TITLE
[질문] google-services.json missing 에러 관련

### DISCUSSION THREAD

**Discussion by wonseok97** [TEAM-DAME/Dame-Android#37] [2. Q&A] [answered]
<img width="1104" alt="image" src="https://user-images.githubusercontent.com/118143620/201555441-b1795508-cd41-4818-ad69-7e7e90111964.png">

다음과 같은 에러가 납니다.

**Comment by onseok** [comment, marked as answer]
google-services.json 파일이 누락되어서 발생하는 오류입니다.
@onseok 에게 요청하여 해당 파일을 app 모듈 루트 디렉토리에 저장하시거나,
초대된 파이어베이스 콘솔의 다메다메 프로젝트 설정에 들어가셔서 직접 google-services.json 파일을 다운로드 할 수 있습니다.
